lms/static/scripts/postmessage_json_rpc/server.ts

```diff
@@ -152,10 +153,8 @@ export class Server {
 
   /**
    * Return true if the provided object is a JSON-RPC request.
-   *
-   * @param {any} request
    */
-  _isJSONRPCRequest(request) {
+  _isJSONRPCRequest(request: any) {
```

Comment:
Should this method be `private`?

```suggestion
  private _isJSONRPCRequest(request: any) {
```

Reply:
Yes, indeed it should be. Thanks for the reminder!